src/modules/blockchain/implementation/gnosis/gnosis-service.js

```diff
@@ -22,9 +22,9 @@ class GnosisService extends Web3Service {
         try {
             const response = await axios.get(this.config.gasPriceOracleLink);
             let gasPrice;
-            if (this.config.name.split(':')[1] === '100') {
+            if (this.getBlockchainId.split(':')[1] === '100') {
                 gasPrice = Number(response.data.result, 10);
-            } else if (this.config.name.split(':')[1] === '10200') {
+            } else if (this.getBlockchainId.split(':')[1] === '10200') {
```

Comment:
```suggestion
            } else if (this.getBlockchainId().split(':')[1] === '10200') {
```

Reply:
This is not sufficient to fix this PR. I've opened a new one